db/migrate/20140615175553_init_schema.rb

```diff
@@ -0,0 +1,871 @@
+class InitSchema < ActiveRecord::Migration
+  def down
+    raise 'Can not revert initial migration'
+  end
+
+  def up
+
+    create_table 'alternative_iris', force: true do |t|
+      t.text     'iri'
+      t.integer  'ontology_id', null: false
+      t.datetime 'created_at',  null: false
+      t.datetime 'updated_at',  null: false
+    end
+
+    add_index 'alternative_iris', ['ontology_id', 'iri'], name: 'index_alternative_iris_on_ontology_id_and_iri', unique: true
```

Comment:
Line is too long. [125/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.